<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeMenu</class>
 <widget class="QDialog" name="changeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="PK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PKchange">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="key2change">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="key3change">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="key4change">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Id </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Марка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Модель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
